examples/notebooks/introduction.ipynb

```diff
@@ -8,10 +8,16 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "[Active learning]( https://en.wikipedia.org/wiki/Active_learning_(machine_learning) ) is method that aims to query new data to be labelled, to improve the machine learning model. We suppose that we have a large pool of unlabelled data. Based on a predefined heuristic strategy, an Oracle will select a certain number of data `num_annotate` that will be annotated and used to re-train the model. Step after step, we should see an improvement in the model's performances. A baseline for the strategy would be to take the data randomly. "
+    "[Active learning]( https://en.wikipedia.org/wiki/Active_learning_(machine_learning) ) is a method that aims to collect and labelled new data in order to improve the machine learning model. Based on a predefined heuristic strategy, a certain number of data will be annotated and used to re-train the model. \n",
+    "In pyrelational, we use: \n",
+    "- an active learning ``strategy``, that query new data points based on specific selection criterion;\n",
+    "- a ``model``, written with pytorch lightning, that takes in input a neural networks and a set of arguments (ie: the number of epochs) used for training;\n",
```

Comment:
this probably should be rephrased:

```suggestion
    "- a ``model manager`` that takes in input an uninstantiated ML model and a set of arguments (ie: the number of epochs) used for training;\n",
```

Reply:
maybe giving more details about the arguments too, e.g. that there are used either to instantiate models or to perform training iterations.

Also might be worth mentioning the Oracle in the list here.